source/leuven/tier2_hardware/wice_hardware.rst

```diff
@@ -1,8 +1,9 @@
 wICE hardware
 ===============
 
-wICE is KU Leuven/UHasselt's latest Tier-2 cluster. It has thin nodes, large memory nodes, interactive nodes and GPGPU nodes.
-This cluster is currently in a pilot phase and will go into production early 2023.
+wICE is KU Leuven/UHasselt's latest Tier-2 cluster. 
+It has thin nodes, large memory nodes, interactive nodes and GPGPU nodes.
```

Comment:
```suggestion
It has thin nodes, large memory nodes, interactive nodes and GPU nodes.
```
(again, to have more consistent naming and because it is to be expected that the GPU nodes contain GPGPUs and not graphics-only-oriented GPU cards)

Reply:
oops. had missed this. fixed now.